main.go

```diff
@@ -232,10 +188,55 @@ func main() {
 		setupLog.Error(err, "unable to create health check")
 		os.Exit(1)
 	}
+}
 
-	setupLog.Info("starting manager", "version", version.Get().String())
-	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
-		setupLog.Error(err, "problem running manager")
+func setupReconcilers(ctx context.Context, mgr ctrl.Manager) {
+	if err := (&controllers.OpenStackClusterReconciler{
+		Client:           mgr.GetClient(),
+		Recorder:         mgr.GetEventRecorderFor("openstackcluster-controller"),
+		WatchFilterValue: watchFilterValue,
+	}).SetupWithManager(ctx, mgr, concurrency(openStackClusterConcurrency)); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "OpenStackCluster")
+		os.Exit(1)
+	}
+	if err := (&controllers.OpenStackMachineReconciler{
+		Client:           mgr.GetClient(),
+		Log:              ctrl.Log.WithName("controllers").WithName("OpenStackMachine"),
```

Comment:
Log should be coming in via the controller-runtime context now.

Reply:
Done